coreos-postinst

```diff
@@ -10,6 +10,7 @@ OEM_MNT="/usr/share/oem"
 
 INSTALL_MNT=$(dirname "$0")
 INSTALL_DEV="$1"
+APP_ID="$2"
```

Comment:
Newer postinst may be called by old update_engine so: `APP_ID="${2:-{e96281a6-d1af-4bde-9a0a-97b76e56dc57}}"`